girder/web_client/src/views/widgets/BrowserWidget.js

```diff
@@ -4,6 +4,9 @@ import _ from 'underscore';
 import HierarchyWidget from '@girder/core/views/widgets/HierarchyWidget';
 import RootSelectorWidget from '@girder/core/views/widgets/RootSelectorWidget';
 import View from '@girder/core/views/View';
+import CollectionModel from '@girder/core/models/CollectionModel';
+import FolderModel from '@girder/core/models/FolderModel';
+import  UserModel from '@girder/core/models/UserModel';
```

Comment:
Extra space after `import`.  You'd think the linter would complain, but it didn't.